tests/unit_test/unit-test-sqs.py

```diff
@@ -0,0 +1,48 @@
+"""
+Copyright (c) 2023, Xgrid Inc, https://xgrid.co
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+"""
```

Comment:
Docstrings: Include docstrings to describe the purpose of the test case and the function being tested. This will help other developers understand the intention of the code.

Reply:
got it